src/ConvexMethod.cpp

```diff
@@ -1,18 +1,48 @@
 #include "obj_planner/ConvexMethod.hpp"
 
+#include <algorithm>
+#include <cmath>
+#include <numeric>
+
+#include "opencv2/imgproc.hpp"
+
+namespace {
+
+constexpr double pi{3.14159265};
+constexpr double epsilon{1e-3};
+
+/// Wrap angle (radians) between [0, 2pi)
+double wrap_to_2pi(double angle) {
+    angle = fmod(angle, 2.0 * pi);
+    if (angle < 0.0) angle += 2.0 * pi;
+    return angle;
+}
+
+/// Only let number reside in [0, 1]
+void clamp_between_0_and_1(double& number) {
+    if (number < 0.0) {
```

Comment:
Not a huge deal, but std::clamp does exist

Reply:
Completely forgot about std::clamp, Ill update to use that instead

Reply:
For now I didn't update this to prioritize getting it tested and working.